pulsar-io/couchbase/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.pulsar</groupId>
+    <artifactId>pulsar-io</artifactId>
+    <version>2.4.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>pulsar-io-couchbase</artifactId>
+  <name>Pulsar IO :: Couchbase</name>
+
+  <properties>
+    <java-client.version>2.7.4</java-client.version>
+    <dcp-client.version>0.22.0</dcp-client.version>
+    <junit-platform.version>1.4.2</junit-platform.version>
+    <junit-jupiter.version>5.4.2</junit-jupiter.version>
+    <assertj.version>3.11.1</assertj.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>pulsar-io-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.couchbase.client</groupId>
+      <artifactId>dcp-client</artifactId>
+      <version>${dcp-client.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.couchbase.client</groupId>
+      <artifactId>java-client</artifactId>
+      <version>${java-client.version}</version>
+    </dependency>
```

Comment:
The main logic is using `dcp-client`, the `java-client` is never used, can it be remove?